examples/camera-controller/commands/interactive/InteractiveCommands.h

```diff
@@ -21,6 +21,9 @@
 #include <commands/clusters/DataModelLogger.h>
 #include <commands/common/CHIPCommand.h>
 #include <commands/common/Commands.h>
+#include <websocket-server/WebSocketServer.h>
+
+#include <websocket-server/WebSocketServer.h>
```

Comment:
duplicated?

Reply:
I will fix this in the next update. Thank you for your eagle eyes. :)